src/OmniSharp.Host/OmniSharpMefBuilder.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using System.Composition.Hosting;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyModel;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OmniSharp.Eventing;
+using OmniSharp.FileWatching;
+using OmniSharp.Mef;
+using OmniSharp.Options;
+using OmniSharp.Roslyn;
+using OmniSharp.Services;
+using OmniSharp.Stdio.Services;
+
+namespace OmniSharp
+{
+    public class OmniSharpMefBuilder
```

Comment:
Please don't prefix class names with `OmniSharp`. That's what namespaces are for.